src/java/fr/paris/lutece/plugins/forms/web/admin/FormResponseJspBean.java

```diff
@@ -0,0 +1,579 @@
+import fr.paris.lutece.portal.util.mvc.commons.annotations.View;
+import fr.paris.lutece.util.url.UrlItem;
+
+@Controller( controllerJsp = "ManageFormResponse.jsp", controllerPath = "jsp/admin/plugins/forms/", right = "FORM_RESPONSE_MANAGEMENT" )
+public class FormResponseJspBean extends AbstractJspBean
+{
+	private static final long serialVersionUID = 1L;
+	
+	// Templates
+	private static final String TEMPLATE_VIEW_STEP = "/admin/plugins/forms/step_view.html";
+	private static final String TEMPLATE_FORM_SUBMITTED = "/admin/plugins/forms/form_submitted_view.html";
+	private static final String TEMPLATE_VIEW_FORM_RESPONSE_SUMMARY = "/admin/plugins/forms/form_response_summary.html";
+
+	// Properties for page titles
+    private static final String PROPERTY_PAGE_TITLE_CREATE_FORM_RESPONSE = "forms.create_form_response.pageTitle";
+    
+    // Markers
+    private static final String MARK_FORM_TITLE = "formTitle";
+    private static final String MARK_LIST_SUMMARY_STEP_DISPLAY = "list_summary_step_display";
+    
+    // Other
+    private Controller _controller = getClass( ).getAnnotation( Controller.class );
+    private FormService _formService = SpringContextService.getBean( FormService.BEAN_NAME );
+    private FormResponseManager _formResponseManager;
+    private Step _currentStep;
+    private StepDisplayTree _stepDisplayTree;
+    private IBreadcrumb _breadcrumb;
+    private final Boolean _frontOffice = false;
+	
+    /**
+     * 
+     * @param request
+     *            The Http request
+     * @return the String
+     * 
+     */
+    @View( value = FormResponseUtils.VIEW_STEP )
+    public String getStepView( HttpServletRequest request )
+    {
+    	String paramInit = request.getParameter( FormsConstants.PARAMETER_INIT );
+        if ( FormResponseUtils.PARAMETER_INIT.equals( paramInit ) )
+        {
+            init( request );
+        }
+
+        int nIdForm = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( nIdForm != FormsConstants.DEFAULT_ID_VALUE && ( _currentStep == null || nIdForm != _currentStep.getIdForm( ) ) )
+        {
+            init( nIdForm );
+        }
+
+        Form form = FormHome.findByPrimaryKey( _currentStep.getIdForm( ) );
+
+        if (  form.getMaxNumberResponse( ) != 0 )
+        {
+            Object lock = FormResponseUtils.getLockOnForm( form );
+            synchronized( lock )
+            {
+            	if ( form.getMaxNumberResponse( ) != 0 )
+                {
+            		Map<Integer, Integer> responsePerFormMap = new HashMap<>( );
+                    int nNumberReponseForm = responsePerFormMap.computeIfAbsent( form.getId( ), FormHome::getNumberOfResponseForms );
+                    if ( nNumberReponseForm >= form.getMaxNumberResponse( ) )
+                    {
+                    	return I18nService.getLocalizedString( FormResponseUtils.MESSAGE_ERROR_NUMBER_MAX_RESPONSE_FORM, getLocale( ) );

```

Comment:
Have you tested the behavior of this return!!!!?
use redirectView please.
Example:
            addError( FormResponseUtils.MESSAGE_ERROR_NUMBER_MAX_RESPONSE_FORM, getLocale( ) );
return redirectView( request, VIEW_......... );